### DISCUSSION TITLE
Overview of "block theme" stuff you can do with classic/hybrid themes

### DISCUSSION THREAD

**Discussion by justintadlock** [WordPress/developer-blog-content#202] [Topic Ideas]
Currently, there is a not a good resource that outlines exactly what "block" features you can use within classic themes. For example, you can use `theme.json`, various `add_theme_support()` opt-ins, block-based template parts, and more.

The proposal is to give an overview of the essential pieces of creating a hybrid theme (i.e., a classic theme that opts into various block features).

This is a middle ground between classic and block that we haven't really embraced much, but it is still a common use case for agencies and developers who still need to use classic things for their projects.

**Comment by bph** [comment]
The topic was approved, discussion is locked and conversation moved to issue 
- #218

**Comment by troychaplin** [comment]
Article has been published https://developer.wordpress.org/news/2024/12/03/bridging-the-gap-hybrid-themes/
